apps/backend/src/services/polling_service.py

```
@@ -83,15 +91,16 @@ async def _polling_loop(self) -> None:
 
     async def _get_devices_to_poll(self) -> List[Device]:
         """Get all devices that should be actively polled"""
-        query = select(Device).where(
-            and_(
-                Device.monitoring_enabled == True,
-                Device.status.in_(["online", "unknown"])  # Don't poll offline devices
+        async with self.session_factory() as db:
+            query = select(Device).where(
+                and_(
+                    Device.monitoring_enabled == True,
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `Device.monitoring_enabled:` for truth checks

Replace with `Device.monitoring_enabled`

(E712)